src/test/java/com/avaje/tests/compositekeys/TestCore.java

```diff
@@ -96,21 +107,32 @@ public void setUp() throws Exception {
     item.setDescription("Another Fancy Car - Customer 2");
     item.setRegion(500);
     item.setType(10);
-    getServer().save(item);
+    server().save(item);
 
-    tx.commit();
+    testCaseTxn.commit();
+  }
+  
+  @After
+  public void tearDown() {
+    if (testCaseTxn != null && testCaseTxn.isActive()) {
+      // transaction left running after the test, rollback it to make
+      // the environment ready for the next test
+      testCaseTxn.rollback();
+    }
   }
 
+  @Test
   public void testFind() {
-    if (isMsSqlServer()) return;
 
-    List<Item> items = getServer().find(Item.class).findList();
+    assumeFalse("Skipping test because key feature in setUp not yet supported for MS SQL Server.", isMsSqlServer());
```

Comment:
do we have to skip the tests, too? Shouldn't an "assumeFalse" in setUp() be sufficient